packages/backend/src/api/controllers/status/StatusController.ts

```diff
@@ -122,6 +128,8 @@ export class StatusController {
     const statuses = await this.reportStatusRepository.getBetween(
       firstHour,
       lastHour,
+      ChainId.ETHEREUM,
```

Comment:
Add TODO and task in Linear to update the status endpoint once we finish the refactoring